app/javascript/components/patient/laboratory/Laboratory.js

```diff
@@ -1,29 +1,21 @@
 import React from 'react';
 import { PropTypes } from 'prop-types';
-import { Button } from 'react-bootstrap';
+import { Button, Dropdown } from 'react-bootstrap';
 import axios from 'axios';
-
 import LaboratoryModal from './LaboratoryModal';
 import reportError from '../../util/ReportError';
+import DeleteDialog from '../../util/DeleteDialog';
 
 class Laboratory extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      showModal: false,
+      showEditModal: false,
```

Comment:
It's not a big deal, but I would love if there were a one-sentence comment somewhere in here quickly describing showEditModal/showDeleteModal and toggleEditModal/toggleDeleteModal because it took me a minute to parse this and I'm working on a parallel task right now.

Reply:
I think the naming is pretty straightforward, we use it all over our code

Reply:
I named mine similarly for the delete history comments, I suggest you do the same for the close contacts

Reply:
^^^ agreed